units/patch_intent.go

```diff
@@ -1071,7 +1071,8 @@ func (j *patchIntentProcessor) buildGithubMergeDoc(ctx context.Context, patchDoc
 		)
 	}
 	if projectRef == nil {
-		return errors.Errorf("project ref for repo '%s/%s' with branch '%s' not found",
+		j.gitHubError = commitQueueDisabled
```

Comment:
coule we tweak this github error? I know you didn't write it but I think it's actually not descriptive for what we're using it for -- specifically, that the merge queue (we should stop using "commit queue" verbiage since that's the legacy feature) isn't enabled on Evergreen for any projects (i'd like to recognize that this original wording was my fault haha)